sale_pricelist_discount/__openerp__.py

```diff
@@ -0,0 +1,44 @@
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 credativ ltd (<http://www.credativ.co.uk>).
+#    All Rights Reserved
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Sale Pricelist Discount',
+    'version': '1.0',
+    'category': 'Sale',
+    'description': """
+This module allows specifying a default percentage discount to apply
```

Comment:
Extract this description to a README.rst and please follow OCA's template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Mention in know issues that it's incompatible with visible discount module (I don't remember now the exact name).